samples/wifi/scan/README.rst

```diff
@@ -0,0 +1,57 @@
+.. _wifi_scan_sample:
+
+Wi-Fi: Scan
+############
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample allows the Nordic Wi-Fi chipset to scan for the Access points.
```

Comment:
I think it is important to state that this sample doesn't pull in the wpa supplicant hence is only good for scanning.

Reply:
That's correct, it is actually 'meant' to do only scanning and we have chosen do so without using wpa_supplicant (so customers have an example of that). So, we could add "without using the wpa_supplicant"?

Reply:
Yes. I think it is important to get the intent behind this sample, e.g., using for location services with minimized code size since supplicant is not available.

Reply:
This change is resolved in another PR.